src/main/java/lt/vu/web/api/dto/postDtos/order/OrderDTO.java

```diff
@@ -0,0 +1,17 @@
+package lt.vu.web.api.dto.postDtos.order;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter @Setter
+public class OrderDTO {
+
+    private SenderDTO sender;
+
+    private SenderDTO recipient;
+
+    private int packageOptionId;
+
+    private String pickUpDate;
```

Comment:
Maybe this could be `Date`? I think jax-rs is smart enough to parse this for us 🤔